bemani/backend/iidx/pendual.py

```diff
@@ -134,10 +134,10 @@ def get_settings(cls) -> Dict[str, Any]:
                     'setting': 'global_shop_ranking',
                 },
                 {
-                    'name': 'Events In Omnimix',
```

Comment:
Why did you get rid of this switch? Every other IIDX supports disabling events in omnimix.

Reply:
I removed it and had forgotten to re-implement it. Will be fixed in the next commit.